### DISCUSSION TITLE
Welcome to Halium-10-Samsung-a50 Discussions!

### DISCUSSION THREAD

**Discussion by Anax-M** [AnaxMorden/Halium-10-Samsung-a50#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  * And I hope you can help me out 

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BroPro40** [comment]
can i help with something?
i do not know any program language, but learning basics of c++

**Reply by sadatdaniel** [reply]
> 
> 
> can i help with something?
> i do not know any program language, but learning basics of c++

In case you don't get reply here, I think the author is also very responsive on [XDA ](https://forum.xda-developers.com/m/mordenstudios.10660331/)forum.

**Reply by Anax-M** [reply]
Sry that I reply now but 
I show where you can help and notify you
Can you write me on Discord or Matrix ( Element.io ) ? There I could contact you better
